### PR TITLE
Feat/get recommendation

### PR DESCRIPTION
**Pull request by lamchiminhmark**
Created and tested logic for creating and combining recommendation in frontend

### CODE REVIEW

File: src/actions/recommenderActions.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const likeEvent = (userId, categories) => dispatch => {
+    const requestOptions  = {
+        method: 'POST',
+        headers: {'Content_Type': 'application/json'},
+        body: JSON.stringify({userId, categories})
+    }
+    return fetch('/rateEvent', requestOptions)
```
Comment (Cpotter2) [inline src/actions/recommenderActions.js:12]: @lamchiminhmark Does returning the fetch call here actually ever dispatch an action to Redux? Or are you not attempting to dispatch an action?

File: src/components/Sidepane/EventDetails.js [hunk 2]
```diff
@@ -48,6 +49,8 @@ class EventDetails extends React.Component {
     const timeString = this.getEventTimeString();
     const locationName = event.location_name || '';
     const locationString = event.address || 'Location unknown.';
+    //TODO: Get real userId
```
Comment (Cpotter2) [inline src/components/Sidepane/EventDetails.js]: @knguyend   Can this now be updated with Firestore access to the real uid? (so we can get rid of this TODO?)

**Review by Cpotter2 — CHANGES_REQUESTED** [review]
Looks good! Just had a couple questions -- see comments.

**Comment by lamchiminhmark** [comment]
Closing PR and opening new one where problems are addressed

**Pull request closed without merging**